next-sitemap.config.js

```diff
@@ -18,13 +18,23 @@ const SITE_URLS = {
 };
 
 /** @type {Array<string>} */
-const PDF_LINKS = ['*/dictionary/pdf/*', '*/kids/stories/pdf/*', '*/alphabet/pdf/*'];
+const PDF_LINKS = [
+  '/dictionary/pdf/*',
+  '/uk/dictionary/pdf/*',
+  '/kids/pdf/*',
+  '/uk/kids/pdf/*',
+  '/kids/stories/pdf/*',
+  '/uk/kids/stories/pdf/*',
+  '/alphabet/pdf/*',
+  '/uk/alphabet/pdf/*',
+  '/pdf/.gitkeep'
```

Comment:
Stále je tam pdf/.gitkeep, mohla bys prosím ověřit, jestli tečka má také speciální význam, libovolný znak? Pokud ano, tak ji můžeme escapovat.